src/components/app.js

```diff
@@ -88,6 +90,14 @@ const useStyles = makeStyles(() => ({
     process: {
         marginLeft: 18,
     },
+    child: {
+        display: 'flex',
+        flexGrow: 10,
```

Comment:
why 10 ? what are the other flexItems and what is their flex-grow value ?

Reply:
There is an empty div in the TopBar.js file line 163 which hinders the display of the button in the right place, since it is at the same level as the div which contains the tab and the button, I think it is better to correct that in the topbar at commons-ui

Reply:
fixed